r-package/inst/include/leapfrog_py.hpp

```diff
@@ -42,32 +42,32 @@ py::dict build_python_output(const OutputState<ModelVariant, real_type> &state,
   int output_years = save_steps.size();
   constexpr auto ss = StateSpace<ModelVariant>();
   constexpr auto dp = ss.dp;
-  py::array_t<real_type, py::array::f_style> p_total_pop({dp.pAG, dp.NS, output_years}, state.dp.p_total_pop.data());
+  py::array_t<real_type, py::array::f_style> p_totpop({dp.pAG, dp.NS, output_years}, state.dp.p_totpop.data());
   py::array_t<real_type, py::array::f_style> births({output_years}, state.dp.births.data());
-  py::array_t<real_type, py::array::f_style> p_total_pop_natural_deaths({dp.pAG, dp.NS, output_years}, state.dp.p_total_pop_natural_deaths.data());
+  py::array_t<real_type, py::array::f_style> p_totpop_natural_deaths({dp.pAG, dp.NS, output_years}, state.dp.p_totpop_natural_deaths.data());
 
   py::dict result;
-  result["p_total_pop"] = p_total_pop;
+  result["p_totpop"] = p_totpop;
   result["births"] = births;
-  result["p_total_pop_natural_deaths"] = p_total_pop_natural_deaths;
+  result["p_totpop_natural_deaths"] = p_totpop_natural_deaths;
```

Comment:
Per the above comment, make this `p_totpop_deaths_natural`?

Also, I think `background` and `natural` are referring to the same thing. Any different recollection on this?

I can't remember which terminology we decided was preferred, but harmonize?